api/src/main/java/org/openmrs/Condition.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import java.util.Date;
+
+/**
+ * This is the condition class.
+ * It records detailed information about a condition, 
+ * problem, diagnosis, or other event, situation or issue.
```

Comment:
You also need to add this https://github.com/openmrs/openmrs-core/commit/ac3c31208a67e2f8264b713f23da3a2252877c37

Reply:
Done